policies/coding-style.md

```diff
@@ -389,7 +408,11 @@ For example:
 
 ## Chapter 8: Commenting
 
-Use the classic "/* ... */" comment markers.  Don't use "// ..." markers.
+Use the classic `/*` ... `*/` comment markers.  Don't use `//` ... markers.
```

Comment:
I think the dots should be within the code markup.

Reply:
Done.